go.mod

```diff
@@ -28,6 +28,6 @@ require (
 	golang.org/x/text v0.8.0 // indirect
 	google.golang.org/genproto v0.0.0-20220804142021-4e6b2dfa6612 // indirect
 	google.golang.org/protobuf v1.28.1 // indirect
-	k8s.io/apimachinery v0.26.0 // indirect
+	k8s.io/apimachinery v0.27.0-rc.0 // indirect
 	k8s.io/component-base v0.26.0 // indirect
```

Comment:
Why is this one v0.26.0, not 0.27.0-rc.0?

Reply:
Why update to a release candidate at all?

1.27 is going to be out next week.

Reply:
The script automatically picked this version.